index.html

```diff
@@ -25,14 +26,14 @@ <h2 class="card__question">
             <button class="card__button-answer" type="button">
               Show answer
             </button>
-            <p class="card__answer">Mark Hamill</p>
+            <p class="card__answer" data-js="answerOne">Mark Hamill</p>
```

Comment:
```suggestion
            <p class="card__answer" data-js="first-answer">Mark Hamill</p>
```
We better should write attribute names with `-`  :) Also just "answer" could be enough for that, or?